code/__DEFINES/traits.dm

```diff
@@ -1350,4 +1350,10 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 ///Trait given when a mob has been tipped
 #define TRAIT_MOB_TIPPED "mob_tipped"
 
+/// That which allows mobs to instantly break down boulders.
```

Comment:
PLEASE mark any edits to traits.dm, TG has a PR moving things around but it also kills the file, ive already tried porting it once and it was a nightmare

Reply:
these are from tg